src/esn.calendar.libs/app/services/event-service.js

```diff
@@ -575,4 +593,7 @@ function calEventService(
       return new CalendarShell(ICAL.Component.fromString(response.data));
     });
   }
+  function getChangeParticipationUrl(jwt) {
+    return `${httpConfigurer.getUrl('/calendar/api/calendars/event/participation')}?jwt=${jwt}`;
```

Comment:
Can we make the `esn-frontend-calendar-public` to use this function also? Right now we have 2 functions that do the exact same thing.

Reply:
yes , we can refactor it